<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="MainTab">
        <property name="currentIndex">
         <number>6</number>
        </property>
        <widget class="QWidget" name="ContractTab">
         <attribute name="title">
          <string>Контракт</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QWidget" name="InternalContractWidget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <layout class="QVBoxLayout" name="ContractLayout"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="ClientTab">
         <attribute name="title">
          <string>Покупатель</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QWidget" name="InternalClientWidget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <layout class="QVBoxLayout" name="ClientLayout"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="DeliveryAgentTab">
         <attribute name="title">
          <string>Перевозчик</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QWidget" name="InternalDeliveryAgentWidget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_12">
               <item>
                <layout class="QVBoxLayout" name="DeliveryAgentLayout"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="ProductTab">
         <attribute name="title">
          <string>Товар</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <widget class="QWidget" name="InternalProductWidget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_15">
               <item>
                <layout class="QVBoxLayout" name="ProductLayout"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="DepartmentTab">
         <attribute name="title">
          <string>Отдел и сотрудники</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_16">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_14">
            <item>
             <widget class="QWidget" name="InternalDepartmentWidget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_18">
               <item>
                <layout class="QVBoxLayout" name="DepartmentLayout"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="DocumentTab">
         <attribute name="title">
          <string>Нормативный документ</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_19">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_17">
            <item>
             <widget class="QWidget" name="InternalDocumentWidget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_21">
               <item>
                <layout class="QVBoxLayout" name="DocumentLayout"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="MaterialTab">
         <attribute name="title">
          <string>Сырье</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_22">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_20">
            <item>
             <widget class="QWidget" name="InternalMaterialWidget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_24">
               <item>
                <layout class="QVBoxLayout" name="MaterialLayout"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>459</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
